proto/osmosis/twap/v1beta1/query.yml

```diff
@@ -0,0 +1,8 @@
+keeper: 
+- path: "github.com/osmosis-labs/osmosis/v10/x/twap"
+- struct: "Keeper"
+GetArithmeticTwap:
+- default_values:
+  - Req.end_time: "time.Now()"
+- func: "twap, err := k.GetArithmeticTwap(Req.pool_id, Req.base_asset, Req.quote_asset, Req.start_time, Req.end_time)"
+- response: "GetArithmeticTwapResponse{twap}"
```

Comment:
I'm assuming that this is just an idea template and is not actually used for code gen yet.

If so, just a thought - we seem to repeat `GetArithmeticTwap` several times. For requests, responses, and the function name we could reuse the `GetArithmeticTwap` from a single place on line 4.

I think it could be useful to keep the configuration as simple as possible and minimize repetitions

Reply:
Yeah just an idea!

So in line 7, thats the name of the internal keeper function

On line 8, I think your right, we could make it accept both `proto struct name` and `registered proto type` (and in the latter case allow just putting `Response{twap}`